docs/data/date-pickers/base-concepts/ComponentExplorerNoSnap.tsx

```diff
@@ -107,14 +107,12 @@ export default function ComponentExplorerNoSnap() {
 
       return isPro
         ? `
-import { ${exportedName} } from '@mui/x-date-pickers-pro'    
-import { ${exportedName} } from '@mui/x-date-pickers-pro/${subPackage}'    
-`
+import { ${exportedName} } from '@mui/x-date-pickers-pro';
+import { ${exportedName} } from '@mui/x-date-pickers-pro/${subPackage}';`
```

Comment:
❤️ But maybe we could also move "`" symbol from L109 to L110 thus avoiding the first empty line when copy-pasting these examples? 🤔

Reply:
I think that the line break is clear: 
<img width="734" alt="Screenshot 2023-01-28 at 02 03 29" src="https://user-images.githubusercontent.com/3165635/215232815-5cd1f747-3c46-4ee2-8c35-1d9367827d6d.png">

Reply:
I was referring to the fact that both the original version as well as your case has leading empty/new line in the pasted content. 🙈 
<img width="616" alt="Screenshot 2023-01-28 at 22 32 22" src="https://user-images.githubusercontent.com/4941090/215294136-a74907a0-90e3-4793-8301-72efb7d36d6d.png">

I was thinking about something like this to avoid this empty line. 🤔 
```diff
diff --git a/docs/data/date-pickers/base-concepts/ComponentExplorerNoSnap.tsx b/docs/data/date-pickers/base-concepts/ComponentExplorerNoSnap.tsx
index 5e102404b..6f27ac442 100644
--- a/docs/data/date-pickers/base-concepts/ComponentExplorerNoSnap.tsx
+++ b/docs/data/date-pickers/base-concepts/ComponentExplorerNoSnap.tsx
@@ -106,15 +106,13 @@ export default function ComponentExplorerNoSnap() {
       const isPro = cleanName.includes('Range');
 
       return isPro
-        ? `
-import { ${exportedName} } from '@mui/x-date-pickers-pro';
+        ? `import { ${exportedName} } from '@mui/x-date-pickers-pro';
 import { ${exportedName} } from '@mui/x-date-pickers-pro/${subPackage}';`
-        : `
-import { ${exportedName} } from '@mui/x-date-pickers/${subPackage}';
+        : `import { ${exportedName} } from '@mui/x-date-pickers/${subPackage}';
 import { ${exportedName} } from '@mui/x-date-pickers';
 import { ${exportedName} } from '@mui/x-date-pickers-pro';`;
     })
-    .join('\n');
+    .join('\n\n');
 
   const content = exportedNames.map((exportedName) => {
     const Component = exportedElements[
```